ci-operator/step-registry/assisted/common/gather/assisted-common-gather-commands.sh

```diff
@@ -20,32 +20,38 @@ cat > gather_logs.yaml <<-EOF
       - "{{ (lookup('env', 'MUST_GATHER') == 'true') | ternary('--must-gather','', '') }}"
       - "{{ (lookup('env', 'GATHER_ALL_CLUSTERS') == 'true') | ternary('--download-all','', '') }}"
   tasks:
-    - name: Gather logs and debug information
+    - name: Gather logs and debug information from all hosts
+      block:
+      - name: Ensure LOGS_DIR exists
+        ansible.builtin.file:
+          path: "{{ LOGS_DIR }}"
+          state: directory
+          mode: '0755'
+
+      - name: Gather sosreport from all hosts
+        ansible.builtin.command: >-
+          sos report --batch --tmp-dir "{{ LOGS_DIR }}" --all-logs
+            -o memory,container_log,filesys,kvm,libvirt,logs,networkmanager,networking,podman,processor,rpm,sar,virsh,yum
+            -k podman.all -k podman.logs
+
+      - name: List swtpm-localca files to a file
+        ansible.builtin.shell: >-
+          ls -ltr /var/lib/swtpm-localca/ >> {{ LOGS_DIR }}/{{ ansible_host }}/libvirt-qemu/ls-swtpm-localca-{{ ansible_host }}.txt
```

Comment:
no, sure if we need to keep that, it is useful?

Reply:
not anymore